SDAPackage/SDA/moving_obstacle.py

```diff
@@ -0,0 +1,26 @@
+from Obstacle import *
+from SDA import Constants
+
+class MovingObstacle(Obstacle)
+	
+
+	def __init__(self, point, radius, height, velocity)
+		super(MovingObstacle, self).__init__(point, Constants.STATIONARY_OBSTACLE_SAFETY_RADIUS)
+
+		self.radius = radius
+		self.height = height
+		self.velocity = velocity
+
+	def get_radius(self)
+
+		return self.radius
```

Comment:
Also, this is not correct. You have a radius, then you have a safety radius. Please see StationaryObstacle for how to write this method properly.